lib/src/components/NxBackButton/__tests__/NxBackButton.test.tsx

```diff
@@ -4,37 +4,47 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import 'jest-enzyme';
+
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+import { within } from '@testing-library/react';
+
 import NxBackButton, { Props } from '../NxBackButton';
 
 describe('NxBackButton', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxBackButton, { href: '/foo' });
+  const quickRender = rtlRender<Props>(NxBackButton, { href: '/foo' }),
+      renderEl = rtlRenderElement<Props>(NxBackButton, { href: '/foo' });
 
-  it('renders a <div> with nx-back-button and tm-back-button classes containing an <a>', function() {
-    expect(getShallowComponent()).toMatchSelector('div.nx-back-button.tm-back-button');
-    expect(getShallowComponent().find('a')).toExist();
+  it('renders a child with the role of link', function() {
+    const el = renderEl()!;
+    expect(within(el).getByRole('link')).toBeInTheDocument();
   });
 
-  it('renders the link with the nx-text-link class', function() {
-    expect(getShallowComponent().find('a')).toHaveProp('href', '/foo');
```

Comment:
Still need to test the `href`

Reply:
> Still need to test the `href`

Thanks, I shouldn't have missed that: [3a17ecb](https://github.com/sonatype/sonatype-react-shared-components/pull/914/commits/3a17ecb2dcef30ec5f0ee19d8e5ba456f7c84efb)